test/user.js

```diff
@@ -0,0 +1,103 @@
+//During the test the env variable is set to test
+process.env.NODE_ENV = 'development';
+
+let chai = require('chai'),
+  chaiHttp = require('chai-http'),
+  server = require('../app'),
+  should = chai.should(),
+  dictum = require('dictum.js'),
+  User = require('../app/models').users;
+
+chai.use(chaiHttp);
+
+describe('User', () => {
+  beforeEach((done) => {
+    User.destroy({where: {}}, (err) => {}).then(done());     
+  });
+
+  /*
+  * Testing the /users (POST) route
+  */
+  describe('/POST users', () => {
+
+    let newUser = {
+      name: 'Kevin',
+      lastName: 'Temes',
+      email: 'kevin.temes@wolox.com.ar',
+      password: '12345678'
+    };
+
+    it('it should successfully POST a user with the given inputs', (done) => {
```

Comment:
the it inside the string seems kind of redundant, you can avoid it and leave `it(' should..`